lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -194,9 +195,9 @@ def Dir.unlink(path)
   # Downloads the contents of a remote directory a
   # local directory, optionally in a recursive fashion.
   #
-  def Dir.download(dst, src, recursive = false, force = true, &stat)
+  def Dir.download(dst, src, recursive = false, force = true, glob = nil, &stat)
```

Comment:
You're totally right, I was thinking in terms of callers who aren't using glob at all. Thanks!